org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/variables/StackFrameScope.java

```diff
@@ -1,63 +0,0 @@
-/*******************************************************************************
-* Copyright (c) 2017 Microsoft Corporation and others.
-* All rights reserved. This program and the accompanying materials
-* are made available under the terms of the Eclipse Public License v1.0
-* which accompanies this distribution, and is available at
-* http://www.eclipse.org/legal/epl-v10.html
-*
-* Contributors:
-*     Microsoft Corporation - initial API and implementation
-*******************************************************************************/
-
-package org.eclipse.jdt.ls.debug.adapter.variables;
-
-import java.util.Objects;
-
-import com.sun.jdi.StackFrame;
-
-public class StackFrameScope {
```

Comment:
This class is changed which changes the implementation of related request handling logic. @andxu please review the new implementation.

Reply:
StackFrameScope and ThreadObjectReference are similar, they could be merged into a neutral VariableProxy. @andxu has agreed that.